code/src/components/TodoItem.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+import { todos } from '../reducers/todos';
+
+import { CustomCheckbox } from './CustomCheckbox';
+import { DeleteContainer, DeleteButton } from '../styledComponents/styled_components';
+import  bin  from '../assets/bin.png';
+
+export const TodoItem = ({ item }) => {
+  const dispatch = useDispatch();
+
+  // Create the onRemoveClicked handler and dispatch action.
+  const onRemoveClicked = () => {
+    dispatch(
+      todos.actions.removeTodo({
+        itemId: item.id
+      })
+    );
+  };
+
+  // Create the onChange handler for handling the done status. Dispatch the action
+  // called setDone and sends with it two pieces of info: the index of the item
+  // and the value of onChangeValue. Done or not. This new value is listened to
+  // by the setDone reducer in todos, and sets the new value in the global/local state.
+  const handleOnChange = () => {
+    dispatch(
+      todos.actions.setDone({
+        itemId: item.id,
+        done: !item.done
+      })
+    );
+  };
+
+  return (
+    <DeleteContainer className={`todo-item ${item.done ? 'done' : ''}`}>
+      <CustomCheckbox 
+        tabIndex='0' 
+        isChecked={item.done}
+        onChangeHandler={handleOnChange}>
+      </CustomCheckbox> 
+      <span className='todo-item-text'>{item.description}</span>
+      <DeleteButton 
+        className='todo-item-remove' 
```

Comment:
I think these classNames are maybe also not needed? 🤔 (unless I am missing a CSS file somewhere...)